Makefile

```diff
@@ -1,6 +1,7 @@
 CC=gcc -g -Wall
 OFLAGS=-lm
-OBJ=src/vic.o src/vic-serial.o src/vic-funcs.o src/vic-exec.o src/vic-var.o
+OBJ=src/vic.o src/vic-serial.o src/vic-funcs.o src/vic-exec.o src/vic-var.o \
```

Comment:
@vargac we need to get rid of these things and figure out how to structure the Makefile better.